privacyidea/static/css/custom-ui.css

```diff
@@ -0,0 +1,6 @@
+#tablemachinelist th.pifilter { white-space: nowrap; }
```

Comment:
The custom-ui.css is a bit misleading.
If this file is part of the project, an update will overwrite any changes in this file.
What would you suggest to make this update safe?

If we also need a rewriteRule to replace this file, I would not call it "custom". This sounds like, the admin could change this file and everything is fine.